addons/website/static/lib/jstz.min.js

```diff
@@ -0,0 +1,2 @@
+/* jstz.min.js Version: 1.0.6 Build date: 2015-11-04 */
```

Comment:
Suggest to include unminified version instead. From [Odoo Guidelines](https://www.odoo.com/documentation/12.0/reference/guidelines.html#javascript-coding-guidelines):

> Never add minified Javascript Libraries